backend.py

```diff
@@ -265,7 +265,37 @@ def create_result(self, std_out, std_err, status="intermediate", files=False):
             for name in filenames:
                 # ToDO: if filesize > 1mb -> generate s3-url
                 file_path = os.path.join(self.tmp_dir, "files", name)
-                mimetype = mime.from_file(file_path)
+                mimetypes.init()
+                mimetypes.add_type("application/vnd.kitware", ".vtu")
+                mimetypes.add_type("application/vnd.kitware", ".vtp")
+                mimetypes.add_type("application/x-vgf", ".vgf")
+                mimetypes.add_type("application/x-vgf3", ".vgf3")
+                mimetypes.add_type("application/x-vgfc", ".vgfc")
+                mime = mimetypes.guess_type(file_path)
+                mimetype = "" 
+                if mime[0] == "text/plain":
+                    fr = open(file_path, "r")
+                    file = fr.readlines()
+                    is_uri = True
+                    regex = re.compile(
+                        r'^(?:http|ftp)s?://' # http:// or https://
+                        r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|' #domain...
+                        r'localhost|' #localhost...
+                        r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})' # ...or ip
+                        r'(?::\d+)?' # optional port
+                        r'(?:/?|[/?]\S+)$', re.IGNORECASE)
```

Comment:
this is also quite static and has not be executed for each file; at least move to __init__ of the ResultStreamer